### ISSUE TITLE
The loss does not decrease during the training process

### ISSUE THREAD

**Issue by pengcheng001** [Sense-GVT/Fast-BEV#27]
![image](https://user-images.githubusercontent.com/28918337/220307567-85815f84-d524-44d3-b936-5595911f79e3.png)

**Comment by ymlab** [comment]
Maybe run the evaluation to see if there is a reasonable accuracy. If not, see if the pretrained model and learning rate need to be adjusted.
